package.json

```diff
@@ -25,31 +25,35 @@
   "dependencies": {
     "assert-plus": "^1.0.0",
     "bs58": "^4.0.1",
-    "cors": "^2.8.4",
-    "crypto-ld": "^3.5.2",
-    "did-method-key": "^0.5.0",
+    "cors": "^2.8.5",
+    "crypto-ld": "^3.7.0",
+    "did-io": "digitalbazaar/did-io#support-v1",
+    "did-method-key": "^0.6.1",
+    "did-veres-one": "veres-one/did-veres-one#support-v1",
     "http-signature-zcap-verify": "^3.0.0",
-    "jsonld-signatures": "^5.0.0",
+    "jsonld-signatures": "^5.0.1",
     "ocapld": "^2.0.0"
   },
   "peerDependencies": {
-    "bedrock": "1.12.1 - 3.x",
-    "bedrock-account": "^2.1.0",
+    "bedrock": "^3.1.0",
+    "bedrock-account": "^2.2.0",
     "bedrock-did-context": "^1.0.0",
-    "bedrock-express": "2.0.8 - 3.x",
+    "bedrock-ledger-context": "^15.0.0",
+    "bedrock-veres-one-context": "^10.0.0",
+    "bedrock-express": "^3.1.0",
     "bedrock-jsonld-document-loader": "^1.0.1",
-    "bedrock-passport": "4.0.2 - 5.x",
-    "bedrock-permission": "^2.5.0",
-    "bedrock-mongodb": "3.x - 6.x",
-    "bedrock-server": "^2.3.2",
-    "bedrock-validation": "^4.1.0",
+    "bedrock-passport": "^5.0.2",
+    "bedrock-permission": "^2.5.4",
+    "bedrock-mongodb": "^6.0.2",
+    "bedrock-server": "^2.6.0",
+    "bedrock-validation": "^4.3.0",
```

Comment:
For my part I would rather you not use whatever tool you're using to change all this stuff.  Although I think these existing ranges have little basis in fact, I don't think it's cool to destroy them automatically like this.

Reply:
No prob. Will revert. (I just suspect this is a cut&paste artifact).

Reply:
@dmitrizagidulin Releasing this package is blocked on your revert.

Reply:
No need to revert, please proceed.